src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -686,16 +680,27 @@ class HttpGoogleServicesDAO(
     val projectNameString = projectName.value
     for {
       _ <- retryWhen500orGoogleError(() => {
-      executeGoogleRequest(billingManager.projects().updateBillingInfo(s"projects/${projectName.value}", new ProjectBillingInfo().setBillingEnabled(false)))
+        executeGoogleRequest(billingManager.projects().updateBillingInfo(s"projects/${projectName.value}", new ProjectBillingInfo().setBillingEnabled(false)))
       })
       - <- retryWhen500orGoogleError(() => {
-        executeGoogleRequest (resMgr.projects ().delete (projectNameString))
+        executeGoogleRequest(resMgr.projects().delete(projectNameString))
       })
     } yield {
       // nothing
     }
   }
 
+  def projectUsageExportBucketName(projectName: RawlsBillingProjectName): String = {
+    s"${projectName.value}-usage-export"
+  }
```

Comment:
Scala - and specifically rawls - idiomatic is to be terse:
```
def projectUsageExportBucketName(projectName: RawlsBillingProjectName) = s"${projectName.value}-usage-export"
```

the compiler can infer the return type of String.